### DISCUSSION TITLE
Ssl Https

### DISCUSSION THREAD

**Discussion by brdarevic** [leokhoa/laragon#392] [General]
I am new to this.
I follow instruction.
Installed.enabled ssl..enable port 443.added laragon cert to trust store.
Installed wordpress website.database etc...
(Ssl)Https:localhost runs and secured but when i change to https:// mywebsite.com (or my ip)i got message on chrome (your connection is not private.Error cert common name invalid).please help. Thx

**Comment by rescenic** [comment]
Common SSL is usually only for domain name, not for IP. 
.com will resolve to top domain .com, I suggest you use different ending.
![https://i.postimg.cc/cHNGWcnk/laragon-vhost-ssl-1.png](https://i.postimg.cc/cHNGWcnk/laragon-vhost-ssl-1.png)
Local SSL Website + ESET Internet Security (it has ESET SSL Filter CA - so there is no error certs):
![https://i.postimg.cc/xdmVBZcb/laragon-vhost-ssl-2.png](https://i.postimg.cc/xdmVBZcb/laragon-vhost-ssl-2.png)

**Comment by Pen-y-Fan** [comment]
It sounds like you enabled security correctly, double check my blog: [How to set up Laragon on a new Windows computer (part 4) - Enable security](https://pen-y-fan.github.io/2022/11/10/how-to-set-up-laragon-on-a-new-windows-computer-part-4/).

I agree with @rescenic do not use **.com** or **.dev** for local development. Use ***.test** or ***.local** 

![Preferences](https://user-images.githubusercontent.com/40126936/216760341-0f6a0014-e46d-4cbc-b8ab-bda878c58d73.png)

Some browsers ask you to accept the self-signed certificate too. Normally you only need to do this once.
